### DISCUSSION TITLE
How to use train model on Google Colab with "CREPE" mode.

### DISCUSSION THREAD

**Discussion by jonnytri53** [voicepaw/so-vits-svc-fork#568] [Q&A]
This is an original code from Google Colab. How can I modify the code similarly like it in the svcg-gui on Windows?. Thank you.

```
#@title Use trained model
#@markdown **Put your .wav file in `so-vits-svc-fork/audio` directory**
from IPython.display import Audio, display
!svc infer drive/MyDrive/so-vits-svc-fork/audio/{NAME}.wav -m drive/MyDrive/so-vits-svc-fork/logs/44k/ -c drive/MyDrive/so-vits-svc-fork/logs/44k/config.json
display(Audio(f"drive/MyDrive/so-vits-svc-fork/audio/{NAME}.out.wav", autoplay=True))
```

**Comment by 34j** [comment]
Read the docs

**Reply by jonnytri53** [reply]
Hi @34j 
I have attempted to go through the readme document, but I couldn't find any information about incorporating the Crepe mode in Google Colab. As I'm relatively new to this, I apologize for any inconvenience. Could you please take a moment to review the original code and guide me on how to make the necessary modifications? I would appreciate it so much.
This is my modification code, but it wasn't working.

#@title Use trained model
#@markdown **Put your .wav file in `so-vits-svc-fork/audio` directory**
from IPython.display import Audio, display
!svc infer **-fm crepe** drive/MyDrive/so-vits-svc-fork/audio/jonnytri.wav -m drive/MyDrive/so-vits-svc-fork/logs/44k/ -c drive/MyDrive/so-vits-svc-fork/logs/44k/config.json
display(Audio(f"drive/MyDrive/so-vits-svc-fork/audio/jonnytri.out.wav", autoplay=True))
Sorry for my inconvenience.

**Reply by 34j** [reply]
That should be correct...but I need more information.
